spec/models/preservation_policy_spec.rb

```diff
@@ -44,11 +44,12 @@
 
   describe '.default_preservation_policy' do
     it 'returns the default preservation policy object' do
-      PreservationPolicy.seed_from_config
+      # db already seeded
       expect(PreservationPolicy.default_preservation_policy).to be_a_kind_of PreservationPolicy
     end
 
     it "raises RecordNotFound if the default policy doesn't exist in the db" do
+      skip('database seeded before running tests; not super-trivial to destroy relevant objects')
       expect { PreservationPolicy.default_preservation_policy }.to raise_error(ActiveRecord::RecordNotFound)
     end
```

Comment:
Same as above - I agree we don't have to keep this test around. So let's just delete it instead of leaving skipped tests around.

Reply:
ditto to above comments.

Reply:
OK, spoke to @jmartin-sul and created #170.